packages/shared/routes/setup/Profile.svelte

```diff
@@ -68,7 +68,7 @@
 
                 const path = await getProfileDataPath($newProfile.name)
                 const machineId = await Platform.getMachineId()
-                const { sendCrashReports } = $initAppSettings
+                const { sendCrashReports } = $initAppSettings ?? {}
```

Comment:
I think it may be better to default to `false` here rather than an empty object `{}` since it is just a boolean option. I believe there's more safety defaults down the line but it could also be done here for clarity.

Reply:
Thanks! currently the error in the console is about destructuring failing, I've just tried with `false` and still getting this error, so for getting work destructuring we might need an object.

Reply:
Ah right, perhaps then `$initAppSettings ?? { sendCrashReports: false }` might work or `const sendCrashReports = $initAppProfile?.sendCrashReports || false`? Probably doesn't matter as long as it's a boolean value rather than `undefined`.

Reply:
yeah both options looks better, will do. Yes it's only a temporal fix meanwhile we add Sentry to mobile, would be nice to have it for alpha release to better identify errors